core/controllers/pages_test.py

```diff
@@ -24,4 +24,3 @@ def test_about_page(self):
         response = self.testapp.get('/about')
         self.assertEqual(response.status_int, 200)
         self.assertEqual(response.content_type, 'text/html')
-        response.mustcontain('Credits', 'License')
```

Comment:
@kevinlee12 This is the back end test that was failing. It specifically checks the About page for these strings, which I've moved to translation keys. I've removed this part of the test for now, but I'd really appreciate your take on this.